openassessment/workflow/models.py

```diff
@@ -613,8 +613,37 @@ class TeamAssessmentWorkflow(AssessmentWorkflow):
     """
     Extends AssessmentWorkflow to support team based assessments.
     """
+    # Only staff assessments are supported for teams
+    STEPS = ['staff']
```

Comment:
nit: since we can ever have one step - why not make a string and called it STAFF_STEP ?

Reply:
It was meant to follow the convention of `AssessmentWorkflow` and allow for extensibility in the future